dockstore-webservice/src/main/resources/migrations.1.7.0.xml

```diff
@@ -281,4 +280,12 @@
             <column name="tosversionacceptancedate" type="timestamp"/>
         </addColumn>
     </changeSet>
+    <changeSet author="gluu (generated)" id="createBlacklistedVersionNames">
+        <createTable tableName="entry_blacklistedversionnames">
+            <column name="entry_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="blacklistedversionnames" type="VARCHAR(255)"/>
```

Comment:
varchar 255 might be pretty easy to hit with version names. I mean, a tweet is now 280 characters?
note, there's no real performance difference between text and varchar(n)
https://www.postgresql.org/docs/9.1/datatype-character.html